question/import_form.php

```diff
@@ -34,10 +34,15 @@
  *
  * @copyright  2007 Jamie Pratt me@jamiep.org
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ * @deprecated since Moodle 4.0 MDL-71608
+ * @see qbank_importquestions\forms\question_import_form
  */
 class question_import_form extends moodleform {
 
     protected function definition() {
+        debugging('Class question_import_form in \core_question\import_form is deprecated,
+        please use qbank_importquestions\forms\question_import_form instead.', DEBUG_DEVELOPER);
```

Comment:
please correct the debug info, i believe its 
qbank_importquestions\forms\question_import_form